### PR TITLE
Some copy edits and minor about page styling updates

### PR DESCRIPTION
**Pull request by ChristopherChudzicki**
### What are the relevant tickets?
None.

### Description (What does it do?)
Two changes:
1. On homepage, Changes "Non-Degree Education" to "Non-Degree Learning" per recommendation from MIT copyeditor 
2. On about page:
    - minor copy edits
    - minor styling updates, mostly color.

**NOTE:** The "About" page figma was recently updated and this does NOT reflect the latest designs.

### Screenshots (if appropriate):
Screenshot 2024-06-21 at 2 01 22 PM
Screenshot 2024-06-21 at 3 08 39 PM
Screenshot 2024-06-21 at 3 08 32 PM

### How can this be tested?
1. View the homepage. It should say "Non-Degree Learning".
2. View the About page. It should look similar to before, with minor copy edits, and should have darkGray2 text.

### CODE REVIEW

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 1]
```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
...
+              academic and professional curriculum.
+            </Typography>
+            <Typography variant="body1">
               MIT's Academic courses, programs, and materials mirror MIT
               curriculum and residential programs, making these available to a
               global audience. Approved by faculty committees, academic content
               furnishes a comprehensive foundation of knowledge, skills, and
               abilities for students pursuing their academic objectives.
-              Renowned for its rigor and challenge, MIT's academic offerings
+              Renowned for its rigor and challenge, MIT’s academic offerings
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:230]: I think all the rest are straight quotes. Might as well keep this one the same?
```suggestion
              Renowned for its rigor and challenge, MIT's academic offerings
```

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 2]
```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
...
-            <BodyText variant="body1">
-              MIT Open organizes non-degree educational opportunities into two
-              categories: academic and professional.
-              <br />
-              <br />
+              Academic and Professional content
+            </Typography>
+            <Typography variant="body1">
+              MIT's non-degree offerings include content developed from MIT's
+              academic and professional curriculum.
```
Comment (pdpinch) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:222]: from https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm?node-id=7566-164686#848720013

```suggestion
              Academic and Professional curriculum.
```

**Review by pdpinch — COMMENTED** [review]
Small text suggestions. 

Do you want to match the colors of https://www.figma.com/proto/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?page-id=107%3A2479&node-id=7566-164686&viewport=-17769%2C102%2C0.39&t=WIvzrzqKIBPGFZkm-1&scaling=scale-down-width&content-scaling=fixed&starting-point-node-id=2197%3A31345 and highlight the "What is MIT Open" section instead of the "About Us" section?

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 3]
```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
...
               current="About Us"
             />
-            <Header variant="h3">About Us</Header>
+            <Typography variant="h3" component="h1">
+              About Us
+            </Typography>
           </BannerContainerInner>
         </BannerContainer>
         <BodyContainer>
           <MissionStatementContainer>
```
Comment (gumaerc) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:120]: Can we get rid of this section? It's not in the design anymore.

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx (hunk 1 above)
Comment (ChristopherChudzicki) [reply frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:230]: Ack. Thanks. Copy paste error from google docs.

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 4]
```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
...
+              further their academic and professional goals.
+            </Typography>
           </MissionStatementContainer>
           <SubHeaderContainer>
             <SubHeaderTextContainer>
               <Typography variant="subtitle1">
                 Our non-degree learning resources empower you to:
               </Typography>
-              <ul>
+              <List>
```
Comment (gumaerc) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:140]: Can we put the mission statement here, above the list and alongside the image, but without the `MissionStatementContainer`?

File: frontends/mit-open/src/pages/AboutPage/AboutPage.tsx [hunk 5]
```diff
@@ -120,126 +111,133 @@ const AboutPage: React.FC = () => {
...
-              Bootcamps, MIT OpenCourseWare, MIT Professional Education, MIT
-              Sloan Executive Education, MIT xPRO, and other departments across
-              MIT.
-            </BodyText>
+            <Typography
+              variant="h4"
+              component="h2"
+              id={WHAT_IS_MIT_OPEN_FRAGMENT_IDENTIFIER}
+            >
+              What is {process.env.SITE_NAME}?
```
Comment (gumaerc) [inline frontends/mit-open/src/pages/AboutPage/AboutPage.tsx:174]: Good call with using `SITE_NAME` here. Can we wrap this section in the white background styles from the design?

**Review by gumaerc — CHANGES_REQUESTED** [review]
I know we said we're not going to update the styles to match the design, but I think we should still get the basic structure into place to match what's current so the content is in the right places as expected. Messing with the image / swoopy graphics at the top can wait until later.

**Comment by ChristopherChudzicki** [comment]
@pdpinch @gumaerc OK, I fixed a few more issues, mostly the white box:
Screenshot 2024-06-21 at 4 05 37 PM

We definitely do need a followup issue for 
- mobile & desktop spacings (I did some of it but i am sure not all of it)
- the background iamge
- the actual image

**Review by gumaerc — APPROVED** [review]
👍

**Comment by pdpinch** [comment]
Thank you Chris. Let's merge this.

**Pull request merged**